### ISSUE TITLE
Add ability to traverse through conversation like warp

### ISSUE THREAD

**Issue by vinayraghu** [unternet-co/client#224]
- Each request response should be easily keyboard navigable 
- Be able to archive parts of the conversation easily from inline
- Be able to continue a conversation from a previous point 

https://github.com/user-attachments/assets/56d830fd-014b-43e3-b2d0-83445166e045
